src/middlewares/isSuperAdmin.js

```diff
@@ -0,0 +1,21 @@
+import Customize from '../helpers/Customize';
+// eslint-disable-next-line import/named
+import { roles, users } from '../database/models';
+
+const isSuperAdmin = async (req, res, next) => {
+  const { roleId, id } = req.user;
+  const result = await roles.findOne({ where: { id: roleId } });
+  const { type } = result.dataValues;
+
+  if (type !== 'super admin') {
```

Comment:
@NiyoEric Isn't it easier to check for the super admin id rather than actual text? E.g instead of `if (type !== 'super admin'` you can have `if (id !== 1)` that is if the super admin id is 1.